### PR TITLE
PCQ

### PR DESCRIPTION
**Pull request by LucaDelBuonoHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRO-7216

### Change description ###
Integrate PCQ into PA

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: app/steps/ui/deceased/details/template.html [hunk 1]
```diff
@@ -85,17 +85,20 @@ <h1 class="govuk-heading-l">{{ content.question }}</h1>
             {
                 classes: "govuk-input--width-2" + (" govuk-input--error" if fields["dob-day"].error or fields["dob-date"].error),
                 name: "day",
-                value: fields["dob-day"].value
+                value: fields["dob-day"].value,
+                autocomplete: "dob-day"
```
Comment (bradley-hmcts) [inline app/steps/ui/deceased/details/template.html:89]: Autocomplete attribute is 'bday-day', similarly with month and year

File: app/steps/ui/deceased/dob/template.html [hunk 2]
```diff
@@ -35,17 +35,20 @@
             {
                 classes: "govuk-input--width-2" + (" govuk-input--error" if fields["dob-day"].error or fields["dob-date"].error),
                 name: "day",
-                value: fields["dob-day"].value
+                value: fields["dob-day"].value,
+                autocomplete: "dob-day"
```
Comment (bradley-hmcts) [inline app/steps/ui/deceased/dob/template.html:39]: Autocomplete attribute is 'bday-day', similarly with month and year

File: app/steps/ui/deceased/dod/template.html [hunk 3]
```diff
@@ -35,17 +35,20 @@
             {
                 classes: "govuk-input--width-2" + (" govuk-input--error" if fields["dod-day"].error or fields["dod-date"].error),
                 name: "day",
-                value: fields["dod-day"].value
+                value: fields["dod-day"].value,
+                autocomplete: "dod-day"
```
Comment (bradley-hmcts) [inline app/steps/ui/deceased/dod/template.html:39]: Autocomplete attribute is 'bday-day', similarly with month and year

File: app/steps/ui/deceased/details/template.html (hunk 1 above)
Comment (LucaDelBuonoHMCTS) [reply app/steps/ui/deceased/details/template.html]: Removed this as it doesn't make sense to have an autocomplete on the DoB of someone who died :)

File: app/steps/ui/deceased/dob/template.html (hunk 2 above)
Comment (LucaDelBuonoHMCTS) [reply app/steps/ui/deceased/dob/template.html]: Removed this as it doesn't make sense to have an autocomplete on the DoB of someone who died :)

File: app/steps/ui/deceased/dod/template.html (hunk 3 above)
Comment (LucaDelBuonoHMCTS) [reply app/steps/ui/deceased/dod/template.html]: Removed this as it doesn't make sense to have an autocomplete on the DoB of someone who died :)

**Pull request merged**